src/background.js

```diff
@@ -3,23 +3,52 @@ import ky from 'ky'
 const prettifyJson = obj => JSON.stringify(obj, null, 2)
 
 // schedules
-chrome.alarms.create('cache-shortcuts', { delayInMinutes: 1, periodInMinutes: 1 })
+chrome.alarms.create('cache-shortcuts', { periodInMinutes: 1 })
 
 // listeners
 chrome.alarms.onAlarm.addListener(async alarm => {
-  await cacheShortcutManifest(alarm)
+  await cacheShortcuts(alarm)
 })
 
-const cacheShortcutManifest = async alarm => {
+/**
+ * @param alarm
+ * @returns {Promise<void>}
+ */
+const cacheShortcuts = async alarm => {
   if (alarm.name !== 'cache-shortcuts') return
 
+  console.log('caching shortcuts')
+
+  const manifest = await cacheShortcutManifest()
+
+  const shortcuts = Object.keys(manifest).map(async urlPattern => {
+    const manifestShortcut = manifest[urlPattern]
+
+    const shortcut = await ky
+      .get(`https://chrome-command-bar.netlify.app/shortcuts/${manifestShortcut}`)
+      .json()
+
+    localStorage.setItem(`ccb/shortcuts/${manifestShortcut}`, prettifyJson(shortcut))
+
+    return shortcut
+  })
+
+  localStorage.setItem('ccb/shortcuts', prettifyJson(await Promise.all(shortcuts)))
```

Comment:
**suggestion:** was thinking of using this as the standard for namespacing items in local storage.  Thoughts?